macOS/DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/VPNAppEventsHandler.swift

```diff
@@ -32,69 +32,79 @@ import Subscription
 ///
 final class VPNAppEventsHandler {
 
-    // MARK: - Legacy VPN Item and Extension
-
-#if NETP_SYSTEM_EXTENSION
-#if DEBUG
-    private let legacyAgentBundleID = "HKE973VLUW.com.duckduckgo.macos.browser.network-protection.system-extension.agent.debug"
-    private let legacySystemExtensionBundleID = "com.duckduckgo.macos.browser.debug.network-protection-extension"
-#elseif REVIEW
-    private let legacyAgentBundleID = "HKE973VLUW.com.duckduckgo.macos.browser.network-protection.system-extension.agent.review"
-    private let legacySystemExtensionBundleID = "com.duckduckgo.macos.browser.review.network-protection-extension"
-#else
-    private let legacyAgentBundleID = "HKE973VLUW.com.duckduckgo.macos.browser.network-protection.system-extension.agent"
-    private let legacySystemExtensionBundleID = "com.duckduckgo.macos.browser.network-protection-extension"
-#endif // DEBUG || REVIEW || RELEASE
-#endif // NETP_SYSTEM_EXTENSION
+    typealias FeatureFlagOverridesPublisher = AnyPublisher<(FeatureFlag, Bool), Never>
```

Comment:
The diff below is awful and it's easier to read this file in the final code.

The changes:
1. Adjustments to make the code testable.
2. Added handling of `applicationDidBecomeActive` where the login items can become disabled.
3. Added the possibility of disabling the VPN in `applicationDidFinishLaunching`.